### PR TITLE
inital Anaconda release

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,41 +1,51 @@
...
   host:
     - python
     - pip
+    - setuptools
+    - wheel
     - cython
-    - numpy
-    - scipy >=0.17
-    - requests
+    - numpy {{ numpy }}
```
Comment (cbouss) [inline recipe/meta.yaml]: scipy, numpy and requests are not needed at build time and should be removed from host.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,41 +1,51 @@
...
     - cython
-    - numpy
-    - scipy >=0.17
-    - requests
+    - numpy {{ numpy }} 
+    - scipy 1.9.3
+    - requests 2.29.0
   run:
     - python
     - {{ pin_compatible('numpy') }}
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - numpy
```

File: recipe/meta.yaml (hunk 1 above)
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml]: done

File: recipe/meta.yaml (hunk 2 above)
Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml]: done

**Review by cbouss — APPROVED** [review]

**Review by ELundby45 — APPROVED** [review]

**Pull request merged**